ios/PresentationLayer/Onboarding/Sources/Onboarding/Login/LoginViewModel.swift

```diff
@@ -16,6 +16,7 @@ final class LoginViewModel: BaseViewModel, ViewModel, ObservableObject {
     
     @Injected private(set) var loginUseCase: LoginUseCase
     @Injected private(set) var trackAnalyticsEventUseCase: TrackAnalyticsEventUseCase
+    @Published private(set) var snackState = SnackState<InfoErrorSnackVisuals>()
```

Comment:
Why don't you put it into the State?

Reply:
Becuase it's state on it's own. It's not one value. And is it safe to put class instance into a struct?

Reply:
Can I close this?